module3/assessments/week1-9715.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: page
+title: Week 3 Assessment
+---
+
+[Follow the instructions on this repo!](https://github.com/turingschool-examples/Launch_Mod3Week1_Assessment)
```

Comment:
Might be useful to put a link to the "solution" repl somewhere so we don't lose it if we want to make a modification at some point. Maybe another good place for an instructor note here with the link? https://replit.com/@Zoe-Farrell/Mod3Week1Assessment#